core/src/main/scala/akka/persistence/cassandra/journal/CassandraEventUpdate.scala

```diff
@@ -22,14 +22,10 @@ private[akka] trait CassandraEventUpdate extends CassandraStatements {
   private[akka] implicit val ec: ExecutionContext
   private[akka] val log: LoggingAdapter
 
-  def psUpdateMessage: Future[PreparedStatement] =
-    session.prepare(updateMessagePayloadAndTags).map(_.setIdempotent(true))
```

Comment:
all statements are idempotent so it is just in the default profile now

Reply:
Nice to remove that boilerplate